doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -65,9 +65,14 @@ In this setup, the underlying internal services follow the same high availabilit
 directly accessing the `mgmt-gateway` internal endpoint, services use the virtual IP specified in the spec.
 This ensures that the high availability mechanism for `mgmt-gateway` is transparent to other services.
 
-Example Configuration for High Availability
+The simplest and recommended way to deploy the mgmt-gateway in high availability mode is by using labels. To
```

Comment:
is `mgmt-gateway` the actual name of the thing?  If so this instance maybe should be in backticks like the one on the next line.  We are admittedly not consistent about this throughout the docs, but one day we should make a pass through to harmonize.

Reply:
`mgmt-gateway` is the name of the management gateway cephadm service